LPS-62874 the panels can't be inside of another portlet

juliocamarero opened the pull request:
Hey Evan, 

I have had to move the html of the panels lower in the html page, however that shouldn't change anything in the UI.

However, that seems to break 2 tests.

Could you please help me? I can't figure out what is going on looking at the poshi summary.

thank you!!

david-truong committed:
LPS-62570 upgrade sass-compiler-jni to 2.0.2

Ithildir committed:
LPS-62570 Use latest

Ithildir committed:
LPS-62570 Fix tests on Windows

Ithildir committed:
LPS-62570 Use latest sass-compiler-ruby

Ithildir committed:
LPS-62570 Unit tests require "modules/frontend/fronted-common-css/tmp"

david-truong committed:
LPS-62570 missing separator

david-truong committed:
LPS-62570 missing separator

Commit:
LPS-62570 updated readme for windows build

thektan committed:
LPS-62559 Change drag handle icon to svg lexicon icon

thektan committed:
LPS-62559 Update drag handle css

ctampoya committed:
LRQA-21587 View Form Workflow status

ctampoya committed:
LRQA-21587 Remove navigator macro

ctampoya committed:
LRQA-21587 Create separate macro to open Publish URL

ctampoya committed:
LRQA-21587 Add use case for LPS-62371 Publish Form with Workflow

Ithildir committed:
LPS-63166 SF

LeventeHudak committed:
LPS-63043 Use colored text only instead of labels and use the narrow progress bar

LeventeHudak committed:
LPS-63043 Descriptive design for export-import-web processes list

LeventeHudak committed:
LPS-63043 Css

ealonso committed:
LPS-62605 Uses <aui:button> tag instead of <a> tag with button styles

ealonso committed:
LPS-62605 Creates new FormNavigatorEntry for layout css

juliocamarero committed:
LPS-62874 the panels can't be inside of another portlet

ealonso committed:
LPS-62605 Style layout set form navigator entries

ealonso committed:
LPS-62605 Style mobile device rules form navigator entry

Commit:
LRQA-22074 Update success path

Commit:
LPS-60355 Register services in extender's context

To avoid having to import Servlet interface in the services' bundle
since it is not using it directly.

Commit:
LPS-60355 Use proper ClassLoader

This includes using the correct AxisServlet. The former one changed the
TCCL to the one in the servletContext bundle, thus missing Axis'
classes.

Commit:
LPS-60355 Proper path

Commit:
LPS-60355 Simplify

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/2162/">test-portal-acceptance-pullrequest(master)</a></p>

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 1 hour 53 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9ef1acf5e1d2c1843391193b45a41c48c351106c">9ef1acf5e1d2c1843391193b45a41c48c351106c</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/2162/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26984/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16250/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-hypersonic/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26887/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16124/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/1</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14568/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/2</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15707/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/3</a></li><li><strike><strong><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14632/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/4</a></strong></strike></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26986/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/5</a></li><li><strike><strong><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16237/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/6</a></strong></strike></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19014/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/7</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16125/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23158/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26522/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23074/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23159/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26987/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14874/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26523/">test-portal-acceptance-pullrequest-batch(master)/modules-unit</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16126/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16127/">test-portal-acceptance-pullrequest-batch(master)/portal-web</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12368/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16128/">test-portal-acceptance-pullrequest-batch(master)/service-builder</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23083/">test-portal-acceptance-pullrequest-batch(master)/tck</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16129/">test-portal-acceptance-pullrequest-batch(master)/unit</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/2194/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2162/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/2162/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>23 Jobs Passed.<br />3 Jobs Failed.</p><pre>Completed with the status of FAILURE.</pre></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14632/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/4</a></h5><h6>Job Results:</h6><p>49 Tests Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=Calendar%23SearchForOtherCalendars,label_exp=!master/14632//testReport/com.liferay.poshi.runner/PoshiRunner/test_Calendar_SearchForOtherCalendars_">PoshiRunner.test[Calendar#SearchForOtherCalendars]</a> - <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/2162/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=Calendar%23SearchForOtherCalendars,label_exp=!master/functional-tomcat-mysql/4/index.html">Poshi Report</a> - <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/2162/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=Calendar%23SearchForOtherCalendars,label_exp=!master/functional-tomcat-mysql/4/summary.html">Poshi Summary</a> - <a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=Calendar%23SearchForOtherCalendars,label_exp=!master/14632//console">Console Output</a></li></ol></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16237/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/6</a></h5><h6>Job Results:</h6><p>49 Tests Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=PGSitesdirectory%23DisplayStyleOptions,label_exp=!master/16237//testReport/com.liferay.poshi.runner/PoshiRunner/test_PGSitesdirectory_DisplayStyleOptions_">PoshiRunner.test[PGSitesdirectory#DisplayStyleOptions]</a> - <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/2162/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=PGSitesdirectory%23DisplayStyleOptions,label_exp=!master/functional-tomcat-mysql/6/index.html">Poshi Report</a> - <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/2162/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=PGSitesdirectory%23DisplayStyleOptions,label_exp=!master/functional-tomcat-mysql/6/summary.html">Poshi Summary</a> - <a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=PGSitesdirectory%23DisplayStyleOptions,label_exp=!master/16237//console">Console Output</a></li></ol></li></ol>

willnewbury committed:
LPS-63095 Move portal-configurator-extender to portal-configuration and rename portal-configuration-extender

willnewbury committed:
LPS-63095 source-formatter.properties

vicnate5 committed:
LRQA-21971 Account for user bar notification badge

vicnate5 committed:
LRQA-21971 Update notification badge view

vicnate5 committed:
LRQA-21971 Create macros to choose workflow task in details

vicnate5 committed:
LRQA-21971 Create new macro for workflow notification navigation

vicnate5 committed:
LRQA-21971 Apply to workflow test

anthony-chu commented:
Hi @juliocamarero, I've made a pull request [here](518) to address the specific panels not being checked. The fix in that ticket should resolve the issues seen in this pull request.

vicnate5 committed:
LRQA-21587 SF

Commit:
LPS-63116 update to use injected reference

Commit:
LPS-63116 move to module

Commit:
LPS-63116 use injected reference

Commit:
LPS-63116 remove audit wiring

mtambara committed:
LPS-63097 Move portal-axis-extender to portal-remote-axis-extender

mtambara committed:
LPS-63097 Move portal-cxf-common to portal-remote-cxf-common

mtambara committed:
LPS-63097 Update java usages

mtambara committed:
LPS-63097 Move portal-cxf-jaxrs-common to portal-remote-cxf-jaxrs-common

mtambara committed:
LPS-63097 Move portal-dependency-manager-tccl to portal-remote-dependency-manager-tccl

mtambara committed:
LPS-63097 Update java usages

mtambara committed:
LPS-63097 Update build.gradle

mtambara committed:
LPS-63097 Update bnd

mtambara committed:
LPS-63097 Move portal-http-tunnel-extender to portal-remote-http-tunnel-extender

mtambara committed:
LPS-60397 Update bnd

mtambara committed:
LPS-63097 Move portal-json-web-service-extender to portal-remote-json-web-service-extender

mtambara committed:
LPS-63097 Update bnd

mtambara committed:
LPS-63097 Update java

mtambara committed:
LPS-63097 Move portal-json-web-service-extender-test to portal-remote-json-web-service-extender-test

mtambara committed:
LPS-63097 Move portal-rest-extender to portal-remote-rest-extender

mtambara committed:
LPS-63097 Update bnd

Ithildir committed:
LPS-63166 Add task descriptions

Ithildir committed:
LPS-63166 Categorize tasks into groups

mtambara committed:
LPS-63097 Move portal-rest-extender-test to portal-remote-rest-extender-test

mtambara committed:
LPS-63097 Update bnd

mtambara committed:
LPS-63097 Move portal-soap-extender to portal-remote-soap-extender

mtambara committed:
LPS-63097 Update bnd

mtambara committed:
LPS-63097 Move portal-soap-extender-test to portal-remote-soap-extender-test

mtambara committed:
LPS-63097 Update bnd

shinnlok committed:
SYNC-1548 Replace anonymousHttpClient with just one that has unlimited connections and apply to GetSyncContext

shinnlok committed:
SYNC-1540 Changed response type

shinnlok committed:
SYNC-1540 Enforce GetUserSitesGroups usage

Commit:
LPS-63116 add project files

Commit:
LPS-60355 We should switch the TCCL classloader when invoking the destroy operation

juliocamarero commented:
Awesome. thank you @anthony-chu!

Commit:
LPS-63132 Remove multiple screen rendering logic from the import layouts UI

Commit:
LRQA-22068 Update getElementText paths

Commit:
LPS-63043 Not needed for export

Commit:
LPS-63043 Second half would be always true since this is exporting

Commit:
LPS-63043 SF

marcellustavares committed:
LPS-62482 Update lang key

marcellustavares committed:
LPS-62482 buildLang

Commit:
LPS-63116 move to module

juliocamarero committed:
LPS-62866 use constant from the application list module

juliocamarero committed:
LPS-62866 we need to know the group in which the category active

juliocamarero committed:
LPS-62866 add the group provider at the portlet level since all the application list taglibs need it

juliocamarero committed:
LPS-62866 use the group from the groupProvider, which may not be the scope group

juliocamarero committed:
LPS-62866 move logic from the jsp to the display context

juliocamarero committed:
LPS-62866 use the right group

juliocamarero committed:
LPS-62866 SF

david-truong committed:
LPS-62570 update windows instructions

mtambara committed:
LPS-63097 Use latest version

Commit:
LPS-63116 add build files

Commit:
LPS-63116 allow for config of filter

Commit:
LPS-63116 audit wiring

Commit:
LPS-63116 remove audit filter

mtambara committed:
LPS-63097 Fake gradle cache

marcellustavares committed:
LPS-62968 Add Pid Mapping

willnewbury committed:
LPS-63095 Update bnd file

marcellustavares committed:
LPS-62968 Rename

marcellustavares committed:
LPS-62968 Update references

Ithildir committed:
LPS-63166 Search inside any src directory, not just the first one

Ithildir committed:
LPS-63166 Extract method

kiyoshilee committed:
LRQA-22123 Rename batches to include more environment information

Ithildir committed:
LPS-63166 Look for test Java files instead of classes

We always add the "startTestableTomcat" dependency, but remove it if there are no test source files. We can't use getCandidateClassFiles() inside the dependsOn closure, because that is executed while the task graph is being generated, which happens before "compileTestIntegrationJava" runs.

Ithildir committed:
LPS-63166 Remove old directories

Ithildir committed:
LPS-63166 Use 1.0.2

Ithildir committed:
LPS-63166 SF

xbrianlee committed:
LRQA-22026 Update test steps

Ithildir committed:
LPS-63166 Use 1.0.139

david-truong committed:
LPS-62570 rely on sass compilers to do all parsing

david-truong committed:
LPS-62570 match our other theme tools and only parse scss files

Commit:
LPS-63116 remove configuration scheduler auditing to scheduler module

Commit:
LPS-63116 move to module

Commit:
LPS-62559 - Use bars instead of move icon

david-truong committed:
LPS-62570 update tests to make sure we aren't parsing css files

Commit:
LPS-60355 We can't know which packages we need to import in advance

david-truong committed:
LPS-62570 relying on parsers allows us to simplify the logic significantly

Ithildir committed:
LPS-62570 Simplify, just concatenate

Ithildir committed:
LPS-62570 Update Maven plugin descriptor

natecavanaugh committed:
LPS-62482 - Source formatting

Ithildir committed:
LPS-62570 SF

natecavanaugh committed:
LPS-62559 - Adding ability to specify a CSS class for the icon

natecavanaugh committed:
LPS-62559 - Source formatting

brianchandotcom committed:
LPS-60355 Rename

kenjiheigel committed:
LRQA-21602 Enable sequential test runs on the PR tester

brianchandotcom committed:
LPS-60355 Simplify?

brianchandotcom committed:
LPS-60355 SF

brianchandotcom committed:
LPS-62605 SF

jbalsas committed:
LPS-63122 Update loader version

jbalsas committed:
LPS-63122 Update gradle cache

brianchandotcom committed:
LPS-62968 SF

Ithildir committed:
LPS-63175 Bundle-SymbolicName is optional

brianchandotcom committed:
LPS-62866 SF

brianchandotcom committed:
LPS-63043 SF

Ithildir committed:
LPS-63175 Bundle-Description and Bundle-Name are both optional

Ithildir committed:
LPS-63175 Bundle-Version is optional

Ithildir committed:
LPS-63175 Fix when Bundle-SymbolicName contains parameters

brianchandotcom committed:
LPS-63116 Hugo SF

brianchandotcom committed:
LPS-63116 VerifyProperties

brianchandotcom committed:
LPS-63116 SF

brianchandotcom committed:
LPS-63116 prep next

brianchandotcom committed:
Revert "LPS-63097 Fake gradle cache"

This reverts commit fcb6f0bf340d6b2a8f03d8b79ab351a8386d5717.

brianchandotcom committed:
LPS-63097 cache

kiyoshilee committed:
LRQA-22123 Allow integration tests to use different database versions

brianchandotcom committed:
Revert "LPS-63166 Use 1.0.139"

This reverts commit 711de491aad7b23919c537a02751cb6cd01ceab8.

brianchandotcom committed:
LPS-62570 prep next

brianchandotcom committed:
LPS-62570 cache

brianchandotcom committed:
LPS-62570 prep next

brianchandotcom committed:
LPS-62570 prep next

brianchandotcom committed:
LPS-62570 prep next use latest

brianchandotcom committed:
LPS-62570 cache

pyoo47 committed:
LRQA-22142 fixJSON call is not needed within a <pre> tag.

pyoo47 committed:
LRQA-22142 Regen

Ithildir committed:
Revert "LPS-62570 Use latest"

This reverts commit abea0665a7ce52f622ad23ae49dbfbd05393c501.

Ithildir committed:
LPS-62570 prep next, use latest

Ithildir committed:
LPS-62570 cache

juliocamarero commented:
https://github.com/brianchandotcom/liferay-portal/pull/36006
